packages/stark-ui/src/modules/dropdown/components/dropdown.component.ts

```diff
@@ -123,6 +140,10 @@ export class StarkDropdownComponent implements OnInit, OnChanges, OnInit {
 		this.logger.debug(componentName + ": component initialized");
 		this.optionsAreSimpleTypes = this.areSimpleTypes();
 		this.setDefaultBlank();
+		if (this.color) {
+			const nativeElement: HTMLElement = this.elementRef.nativeElement;
+			nativeElement.classList.add("stark-" + this.color);
```

Comment:
We should try to use the Angular's [Renderer2](https://angular.io/api/core/Renderer2#addClass) for this instead of directly manipulating the DOM ;)